test/associations/staticEager.test.js

```diff
@@ -0,0 +1,167 @@
+        var emp = new Employee();
+        var work = new Works();
+        assert.deepEqual(emp.associations, ["works"]);
+        assert.deepEqual(work.associations, ["employee"]);
+    });
+
+    it.describe("load associations", function (it) {
+
+        it.beforeEach(function () {
+            return comb
+                .when([
+                    Employee.remove(),
+                    Works.remove()
+                ])
+                .chain(function () {
+                    return new Employee({
+                        lastName: "last" + 1,
+                        firstName: "first" + 1,
+                        midInitial: "m",
+                        gender: gender[1 % 2],
+                        street: "Street " + 1,
+                        city: "City " + 1,
+                        works: [{
+                            companyName: "Google",
+                            salary: 100000
+                        },{
+                            companyName: "Alphabet",
+                            salary: 100000
+                        }]
+                    }).save();
+                }).chain(function () {
+                    return new Employee({
+                        lastName: "Skywalker",
+                        firstName: "Luke",
+                        midInitial: "m",
+                        gender: gender[1 % 2],
+                        street: "Street " + 1,
+                        city: "City " + 1,
+                        works: {
+                            companyName: "C2FO",
+                            salary: 200000
+                        }
+                    }).save();
+                });
+
+        });
+
+        it.should("when querying", function () {
+            return comb
+                .when([Employee.eager('works').one(), Works.eager('employee').one()])
+                .chain(function (res) {
+                    var emp = res[0], work = res[1];
+                    var empWorks = emp.works, worksEmp = work.employee;
+                    assert(emp.works[0].id, work.id);
+                    assert(work.employee.id, emp.id);
+                });
+        });
+
+        it.should("when querying with filtering", function () {
+            return Employee.eager('works').filter({lastName: "Skywalker"}).one()
+                .chain(function (emp) {
+                    assert(emp.id, emp.works[0].employeeId);
+                });
+        });
+
+        it.should("and load other eager queries", function () {
+            return Employee.eager('works').eager({
+                who: function(emp) {
+                    return Employee.findById(emp.id);
+                }
+            }).one()
+                .chain(function (emp) {
+                    assert(emp.id, emp.works[0].employeeId);
+                    assert(emp.id, emp.who.id);
+                });
+        });
+
+    });
+
+    it.describe("dataset loading", function (it) {
+
```

Comment:
This test probably doesn't belong here. Ill look for a better spot. The queries test file though doesn't actually run any queries, so I was looking for a spot that already has DB connections setup.